rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/toolWindow/UnityToolWindowManager.kt

```diff
@@ -1,16 +1,17 @@
 package com.jetbrains.rider.plugins.unity.toolWindow
 
+import com.intellij.openapi.diagnostic.Logger
 import com.intellij.openapi.project.Project
+import com.jetbrains.rd.platform.util.idea.ProtocolSubscribedProjectComponent
 import com.jetbrains.rd.util.reactive.whenTrue
-import com.jetbrains.rdclient.util.idea.ProtocolSubscribedProjectComponent
 import com.jetbrains.rider.model.rdUnityModel
 import com.jetbrains.rider.plugins.unity.UnityHost
 import com.jetbrains.rider.projectView.solution
-import com.jetbrains.rider.util.idea.getLogger
 
 class UnityToolWindowManager(project: Project) : ProtocolSubscribedProjectComponent(project) {
+
     companion object {
-        private val myLogger = getLogger<UnityToolWindowManager>()
+        private val myLogger = Logger.getInstance(UnityToolWindowManager::class.java)
```

Comment:
IJ convention is `val LOG = logger< UnityToolWindowManager>()`, we'll eventually migrate to that (rdclient `val logger` is polluting import completion for idea project), mention this here just to say that there's a function `logger` in platform util 😸

Reply:
Cool, good to know. I'll migrate these in a second PR. I've made a note of a few things that need fixing that aren't directly related to upgrading the SDK.